### PR TITLE
Proposed M3 modeling strategy.

### PR DESCRIPTION
**Pull request by matthewclegg**

### CODE REVIEW

File: src/simulator/tests/m3_strategy_test.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        budget: PrivacyBudget,
+        privacy_budget_split: float = 0.5,
+        max_frequency: int = 1,
+    ) -> ReachPoint:
+        rp1 = self.curve.by_spend([spends[0]], max_frequency=max_frequency)
+        rp2 = self.curve.by_spend([spends[1]], max_frequency=max_frequency)
+
+        imps = rp1.impressions[0] + rp2.impressions[0]
+        spend = rp1.spends[0] + rp2.spends[0]
+        freqs = [rp1.reach(i) + rp2.reach(i) for i in range(1, max_frequency + 1)]
```
Comment (jiayu-google) [inline src/simulator/tests/m3_strategy_test.py:60]: So this fake surface is in fact a non-overlap surface, i.e., a plane. LMK if I understand correctly.

**Review by jiayu-google — APPROVED** [review]
Thank you Matthew! It is quite clear and I didn't catch any mistake.

File: src/simulator/tests/m3_strategy_test.py (hunk 1 above)
Comment (matthewclegg) [reply src/simulator/tests/m3_strategy_test.py:60]: It's true that the audiences for the two reach curves are non-overlapping, but the surface is not a plane.

Comment (jiayu-google) [reply src/simulator/tests/m3_strategy_test.py:60]: You are right.

**Review by jiayu-google — APPROVED** [review]

**Review by RonPinelliMRC — APPROVED** [review]
MRC approving based on cursory review after confirming non-independent code review on an exception basis. Experimental code will require independent review prior to publication of or reliance on results generated.

**Pull request merged**